common/rulesys.cpp

```diff
@@ -24,462 +24,451 @@
 #include <cstring>
 #include <fmt/format.h>
 
-/*
-	Commands:
-	#rules:
-	current -> lists current set name
-	switch (set name) -> change set in the DB, but dont reload
-	load (set name) -> load set into this zone without changing the world
-	wload (set name) -> tell world and all zones to load this rule set
-	store [set name] -> store the current rules in this zone to the set (or active if not specified)
-	reset -> reset all rule values to their defaults.
-	list [catname]
-	set (cat) (rule) (value)
-	values [catname] -> show the values of all rules in the specified category/
-*/
+#include "../common/repositories/rule_sets_repository.h"
+#include "../common/repositories/rule_values_repository.h"
 
-const char *RuleManager::s_categoryNames[_CatCount+1] = {
-	#define RULE_CATEGORY(catname) \
-		#catname ,
+std::string RuleManager::s_categoryNames[_CatCount + 1] = {
```

Comment:
Changing this to std::string could cause static initialization problems since RuleManager is a singleton. Its constructor might try to access `s_categoryNames` before the strings are initialized. I'd recommend reverting this change and leaving it as is or investigating changing this in a separate patch